projects/ngrx.io/content/guide/signals/signal-method.md

```diff
@@ -44,27 +44,27 @@ export class NumbersComponent {
 ## Automatic Cleanup
 
 `signalMethod` uses an `effect` internally to track the Signal changes.
-By default, the `effect` runs in the injection context of the caller. In the example above, that is `NumbersComponent`. That means, that the `effect` is automatically cleaned up when the component is destroyed.
+By default, the `effect` runs in the injection context of the caller. In the example above, that is the `Numbers` component. That means, that the `effect` is automatically cleaned up when the component is destroyed.
 
 If the call happens outside an injection context, then the injector of the `signalMethod` is used. This would be the case, if `logDoubledNumber` runs in `ngOnInit`:
 
 ```ts
 @Component({ /* ... */ })
-export class NumbersComponent implements OnInit {
+export class Numbers implements OnInit {
   readonly logDoubledNumber = signalMethod<number>((num) => {
     const double = num * 2;
     console.log(double);
   });
 
   ngOnInit(): void {
     const value = signal(2);
-    // 👇 Uses the injection context of the `NumbersComponent`.
+    // 👇 Uses the injection context of the `Numbers` component.
     this.logDoubledNumber(value);
   }
 }
 ```
 
-Even though `logDoubledNumber` is called outside an injection context, automatic cleanup occurs when `NumbersComponent` is destroyed, since `logDoubledNumber` was created within the component's injection context.
+Even though `logDoubledNumber` is called outside an injection context, automatic cleanup occurs when the `Numbers` component is destroyed, since `logDoubledNumber` was created within the component's injection context.
```

Comment:
```suggestion
Even though `logDoubledNumber` is called outside an injection context, automatic cleanup occurs when the `NumbersDoubler` component is destroyed, since `logDoubledNumber` was created within the component's injection context.
```